package.json

```diff
@@ -18,6 +18,7 @@
   "devDependencies": {
     "@types/eslint": "^8.37.0",
     "@types/jest": "^29.5.0",
+    "@types/node": "^20.11.0",
```

Comment:
@kurtextrem I dont think we need this? Was it erroring?

Reply:
yep it was for me. Maybe because I'm using pnpm?